public/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <link rel="icon" href="./favicon.ico" />
+        <link
```

Comment:
Hey dude, please move this from static stylesheets to node_modules:
- https://www.npmjs.com/package/@fortawesome/fontawesome-free

You can check out the usage of it in this project: 
- https://github.com/creativetimofficial/argon-dashboard-react
- https://github.com/creativetimofficial/argon-dashboard-react/blob/master/src/index.js

Reply:
It is easier to maintain and update if they are in node_modules.